src/throttler.service.ts

```diff
@@ -8,26 +8,33 @@ import { ThrottlerStorage } from './throttler-storage.interface';
  */
 @Injectable()
 export class ThrottlerStorageService implements ThrottlerStorage, OnApplicationShutdown {
-  private _storage: Record<string, ThrottlerStorageOptions> = {};
+  private _map: Map<string, ThrottlerStorageOptions> = new Map();
```

Comment:
I think we should keep the old name `_storage` instead of rebranding it. The fact that we are using a Map is just a implementation detail